auth-service/src/main/java/com/g1/mychess/auth/client/EmailServiceClient.java

```diff
@@ -30,7 +30,7 @@ public void sendVerificationEmail(EmailRequestDTO emailRequestDTO) {
     public void sendPasswordResetEmail(EmailRequestDTO emailRequestDTO) {
         try {
             webClient.post()
-                    .uri("/api/v1/email/send-password-reset")
+                    .uri("/api/v1/email/password-resets")
```

Comment:
I think this should be password-reset instead of resets, resets make it sound plural which is a weird way of naming the endpoint in my opinion

Reply:
Thank you for the feedback! I understand your point about using "password-reset" in the endpoint name to keep it singular and avoid any confusion. I’d love to hear if others have thoughts on this as well, especially if there are preferences or conventions around singular vs. plural terms in endpoint naming.

Looking forward to any additional perspectives!

Reply:
Great discussion! I think it is important to use standardised nomenclature.

Refer to: https://restfulapi.net/resource-naming/

Reply:
I have renamed it to password-recovery, which is a noun.

Reply:
I have merged the pull request.